spec/lib/data_update_scripts/migrate_relevant_fields_from_users_to_users_settings_spec.rb

```diff
@@ -3,7 +3,7 @@
   "lib/data_update_scripts/20210423155327_migrate_relevant_fields_from_users_to_users_settings.rb",
 )
 
-describe DataUpdateScripts::MigrateRelevantFieldsFromUsersToUsersSettings do
+describe DataUpdateScripts::MigrateRelevantFieldsFromUsersToUsersSettings, sidekiq: :inline do
```

Comment:
This is the part that tells the Sidekiq jobs to run inline — the interface for the change in `rails_helper.rb`. No need to add `Sidekiq::Testing.inline { ... }` blocks in every test anymore.

Reply:
It looks like there's a cleanup possible post merge - the honeycomb middleware spec, follow model spec, comments create request spec, and a shared worker example for "enqueues on correct queue" all use these explicitly

I think that's not difficult and can take care of it.